npc/instances/EndlessTower.txt

```diff
@@ -401,895 +403,795 @@ function	script	F_Tower_Monster	{
+		areamonster(.@map$, 93, 9, 103, 45, _("Incarnation of Morroc"), MOROCC_4, 5, .@label$);
 		break;
 	case 98:
-		areamonster .@map$,181,9,191,45,"Kathryne Keyron",1639,2,.@label$;
-		areamonster .@map$,181,9,191,45,"Seyren Windsor",1634,2,.@label$;
-		areamonster .@map$,181,9,191,45,"Margaretha Sorin",1637,2,.@label$;
-		areamonster .@map$,181,9,191,45,"Eremes Guile",1635,2,.@label$;
-		areamonster .@map$,181,9,191,45,"Howard Alt-Eisen",1636,2,.@label$;
-		areamonster .@map$,181,9,191,45,"Cecil Damon",1638,2,.@label$;
+		areamonster(.@map$, 181, 9, 191, 45, _("Kathryne Keyron"), KATRINN, 2, .@label$);
+		areamonster(.@map$, 181, 9, 191, 45, _("Seyren Windsor"), SEYREN, 2, .@label$);
+		areamonster(.@map$, 181, 9, 191, 45, _("Margaretha Sorin"), MAGALETA, 2, .@label$);
+		areamonster(.@map$, 181, 9, 191, 45, _("Eremes Guile"), EREMES, 2, .@label$);
+		areamonster(.@map$, 181, 9, 191, 45, _("Howard Alt-Eisen"), HARWORD, 2, .@label$);
+		areamonster(.@map$, 181, 9, 191, 45, _("Cecil Damon"), SHECIL, 2, .@label$);
 		break;
 	case 99:
-		for(.@i = 1; .@i<=7; ++.@i) {
-			.@mob_ran = rand(1,30);
+		for (.@i = 1; .@i <= 7; ++.@i) {
+			.@mob_ran = rand(1, 30);
 			if (.@mob_ran == 1)
-				areamonster .@map$,267,9,277,45,"Kathryne Keyron",1639,2,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Kathryne Keyron"), KATRINN, 2, .@label$);
 			else if (.@mob_ran == 2)
-				areamonster .@map$,267,9,277,45,"Seyren Windsor",1634,2,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Seyren Windsor"), SEYREN, 2, .@label$);
 			else if (.@mob_ran == 3)
-				areamonster .@map$,267,9,277,45,"Margaretha Sorin",1637,2,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Margaretha Sorin"), MAGALETA, 2, .@label$);
 			else if (.@mob_ran == 4)
-				areamonster .@map$,267,9,277,45,"Eremes Guile",1635,2,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Eremes Guile"), EREMES, 2, .@label$);
 			else if (.@mob_ran == 5)
-				areamonster .@map$,267,9,277,45,"Howard Alt-Eisen",1636,2,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Howard Alt-Eisen"), HARWORD, 2, .@label$);
 			else if (.@mob_ran == 6)
-				areamonster .@map$,267,9,277,45,"Cecil Damon",1638,2,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Cecil Damon"), SHECIL, 2, .@label$);
 			else if (.@mob_ran == 7)
-				areamonster .@map$,267,9,277,45,"Incarnation of Morroc",1918,5,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Incarnation of Morroc"), MOROCC_1, 5, .@label$);
 			else if (.@mob_ran == 8)
-				areamonster .@map$,267,9,277,45,"Incarnation of Morroc",1919,5,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Incarnation of Morroc"), MOROCC_2, 5, .@label$);
 			else if (.@mob_ran == 9)
-				areamonster .@map$,267,9,277,45,"Incarnation of Morroc",1920,5,.@label$;
+				areamonster(.@map$, 267, 9, 277, 45, _("Incarnation of Morroc"), MOROCC_3, 5, .@label$);
 			else if (.@mob_ran == 10)
-				areamonster .@map$,267,9,277,45,"Incarnation of Morroc
```

Comment:
This line looks very long - while it's true that we don't enforce a maximum line length. I think it'd be more comfortable to handle if it was split into multiple lines (i.e. 2 ~ 10 on the first line; 11 ~ 20 on the second, etc)

```
setarray(.@x[2], 136, 224, 310, 395,  52, 136, 224, 310, 395, //  2-10
             52, 136, 224, 310, 395,  52, 136, 224, 310, 395, // 11-20
             52, 136, 224, 310, 395,   0, 136, 224, 310, 395, // 21-30
             52, 136, 224, 310, 395,  52, 136, 224, 310, 395, // 31-40
             52, 136, 224, 310, 395,  52, 136, 224, 310, 395, // 41-50
              0, 136, 224, 310, 395,  52, 136, 224, 310, 395, // 51-60
             52, 136, 224, 310, 395,  52, 136, 224, 310, 395, // 61-70
             52, 136, 224, 310, 395,   0, 136, 224, 310, 395, // 71-80
             52, 136, 224, 310, 395,  52, 136, 224, 310, 395, // 81-90
             52, 136, 224, 310, 395,  52, 136, 224, 309);     // 91-99
```

Reply:
Sure, I will do like that, then.